### ISSUE TITLE
No samples for lineEnds

### ISSUE THREAD

**Issue by fesoliveira014** [HaxeCheckstyle/haxe-formatter-docs#3]
I was going through the documentation online and I noticed that there is no samples for lineEnds. The source code tries to fetch them from the docs, but there is no lineEnds directory under samples.

**Comment by AlexHaxe** [comment]
yes, I haven't written any samples for lineEnds and there is a lot of other stuff missing. feel free to send pull requests with new samples for whatever you feel deserves one.
